pkg/migrations/migration_version.go

```diff
@@ -61,11 +61,11 @@ func Read(dbPath string) (*MigrationVersion, error) {
 
 // SetCurrent update the database migration version of a database directory.
 func SetCurrent(dbPath string) {
-	if curr, err := Read(dbPath); err != nil || curr.MainVersion != version.GetMainVersion() || curr.SeqNum != CurrentDBVersionSeqNum() {
+	if curr, err := Read(dbPath); err != nil || curr.MainVersion != version.GetMainVersion() || curr.SeqNum != LastRocksDBVersionSeqNum() {
```

Comment:
I think this is just generally safer if we say > LastRocksDBVersion?

Reply:
@connorgorman well maybe but this gets called on other clones and we have to account for the LastPersistedTime so we can properly go from rocks -> Postgres -> Rocks -> Postgres.  Take a look at #4511 where I had to make some additional changes to satisfy the case of  rocks -> Postgres -> Rocks -> Postgres all on the same software version as in that instance we need to ensure the migrations run again